common/journal_entries/dvg_beornia.txt

```diff
@@ -0,0 +1,829 @@
+﻿

```

Comment:
[10:25:22][gamedatabase.h:314]: Overriding entry 'je_ameriga_conquer' for database 'common/journal_entries' in ' file: common/journal_entries/dvg_beornia.txt line: 490'
[10:25:22][gamedatabase.h:314]: Overriding entry 'je_world_conquer' for database 'common/journal_entries' in ' file: common/journal_entries/dvg_beornia.txt line: 584'